src/grammar/query.rs

```diff
@@ -131,6 +137,19 @@ fn parse_column_expr(p: &mut Parser) {
     p.finish();
 }
 
+fn parse_as_alias(p: &mut Parser) {
+    p.start(SyntaxKind::Alias);
+    p.expect(T![as]);
+    p.expect(T![quoted_ident]);
+    p.finish();
+}
+
+fn parse_alias(p: &mut Parser) {
+    p.start(SyntaxKind::Alias);
+    p.expect(T![quoted_ident]);
+    p.finish()
+}
```

Comment:
Could we combine these functions into a single `parse_alias` by using `p.eat(T![as])`?